q2_gunc/assets/js/scripts.js

```diff
@@ -236,6 +241,9 @@ document.addEventListener('DOMContentLoaded', function() {
       createSummaryPlot(filteredData, reverseY);
     }
   }
+  // Determine if this is FeatureData[MAG] (dereplicated MAGs) or SampleData[MAGs] (per-sample MAGs)
+  const isFeatureDataMAG = sampleKeys.includes("") && sampleKeys.length === 1;
```

Comment:
Do we need to calculate this again here? We already do on line 65 - perhaps this can just be passed to this function here?